index.js

```diff
@@ -30,6 +31,7 @@ function HTMLReactParser(html, options) {
 
 HTMLReactParser.domToReact = domToReact;
 HTMLReactParser.htmlToDOM = htmlToDOM;
+HTMLReactParser.attributesToProps = attributesToProps;
```

Comment:
Can you add a unit test and update types?

Reply:
OK